clients/java/src/main/java/io/camunda/client/impl/command/JobUpdateCommandImpl.java

```diff
@@ -24,6 +24,7 @@
 import io.camunda.client.api.response.UpdateJobResponse;
 import io.camunda.client.impl.http.HttpCamundaFuture;
 import io.camunda.client.impl.http.HttpClient;
+import io.camunda.client.impl.response.UpdateJobResponseImpl;
```

Comment:
💭  Also here we have a bit of incosistent naming `JobUpdateCommand` should better be `UpdateJobCommnd`. It seems sometimes we have the verb first and sometimes the noun 🤔